### ISSUE TITLE
A11y_code.visualstudio.com_Docs Azure_Try Azure App Service_ScreenReader: Screen reader announce "Microsoft Azure" link as "Utility Navigation" link which is incorrect.

### ISSUE THREAD

**Issue by Charanrajkb** [microsoft/vscode-docs#2799]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](url)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.3.1, MAS 2.4.4

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/tryappservice/](url)
or
Follow below steps:
1. Launch the application Url : [https://code.visualstudio.com](url)
2. Navigate to "Docs" link.
3. From left navigation, select "Azure'.
4. Navigate to "Try Azure App Service" link.
5. Start Narrator.
6. Navigate using the Tab Key till the Microsoft azure link.
7. Observe the narrator reading.

**Actual:**
Screen reader announce "Microsoft Azure" link as "Utility Navigation" link which is incorrect.

**Issue is observed in Edge + Narrator and Chrome + NVDA.** 

**Expected:**
Screen reader should announce the what ever the link name present on the screen.
Screen reader must read as Microsoft Azure link.

**Recommendations**:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
Screen reader user will not understand the link purpose  if the link is read incorrect.

**MAS Reference:**
MAS 1.3.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)
MAS2.4.4:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}](url)

**Attachment for Reference**:
[A11y_code.visualstudio.com_Docs Azure_Try Azure App Service_ScreenReader.pptx](https://github.com/microsoft/vscode-docs/files/3320463/A11y_code.visualstudio.com_Docs.Azure_Try.Azure.App.Service_ScreenReader.pptx)

**Comment by bowdenk7** [comment]
Fixed by https://github.com/microsoft/TryAppServiceClient/commit/3d48ef2aa6f501680adbd6148f5427884332f001
